src/Akka.Streams.Kafka.Tests/Integration/PlainSourceIntegrationTests.cs

```diff
@@ -28,13 +29,14 @@ public PlainSourceIntegrationTests(ITestOutputHelper output, KafkaFixture fixtur
         }
     
 
-        private TestSubscriber.Probe<string> CreateProbe(ConsumerSettings<Null, string> consumerSettings, ISubscription sub)
+        private Tuple<IControl, TestSubscriber.Probe<string>> CreateProbe(ConsumerSettings<Null, string> consumerSettings, ISubscription sub)
```

Comment:
LGTM - I'm sure we'll switch this to `ValueTuple` once we've made that change on Akka.Streams itself in the next v1.4.0-beta release.